### DISCUSSION TITLE
Bam file DNE

### DISCUSSION THREAD

**Discussion by ashleethomson** [stjude/CICERO#50] [Q&A] [answered]
Hi, hoping I can get some help with an error that keeps popping up when running CICERO through docker. 

`ERROR: Bamfile '../starfusion/Aligned.out.bam' DNE`

I used STAR to generate the BAM file, and I've tried different BAM files in different locations, both sorted and unsorted, but I keep receiving this error. I'm assuming DNE means Does Not Exist. I can't actually find any official documentation regarding it. First-time CICERO user so hoping it's a simple fix. Any advice would be much appreciated and thanks in advance!

**Comment by adthrasher** [comment, marked as answer]
Can you share the full command you are using to invoke CICERO? It appears that you are providing a local path, which is unavailable in the container.

**Reply by ashleethomson** [reply]
Oh ok, then that's probably the issue haha. I'm currently working on a Google Cloud VM.

`docker run -v CICERO-output:/reference stjude/cicero:1.4.0 Cicero.sh [-n 8] -b ../starfusion/Aligned.out.bam -g ../reference-GRCh38/Homo_sapiens/GRCh38_no_alt -r ../reference-GRCh38/Homo_sapiens/GRCh38_no_alt -o CICERO-output [-j ../starfusion/Chimeric.out.junction] [-p true] [-s 2] [-t 200000] [-c 3]`

**Reply by adthrasher** [reply]
With Docker, you need to provide mount instructions to make the local paths available inside the container. These are specified with the `-v` argument to docker. You also don't want to include the brackets (`[]`) for the arguments, that just specifies that the arguments are optional. 

Your command would look something like this where `reference_dir`, `starfusion_dir`, and `output_dir` are replaced with the appropriate paths on your file system. 

`docker run -v reference_dir:/reference -v starfusion_dir:/starfusion -v output_dir:/CICERO-output stjude/cicero:1.4.0 Cicero.sh -n 8 -b /starfusion/Aligned.out.bam -g GRCh38_no_alt -r /reference -o /CICERO-output -j /starfusion/Chimeric.out.junction -p  -s 2 -t 200000 -c 3`

**Reply by ashleethomson** [reply]
Thank you! I thought it might be something simple like that. I'm still new to Docker as well. Very much appreciate the help!
